README.md

```diff
@@ -1,34 +1,61 @@
 # DCASE2022-data-generator
-Data generator for creating synthetic audio mixtures suitable for DCASE Challenge 2022 Task 3
+Data generator for creating synthetic audio mixtures suitable for DCASE Challenge Task 3
 
 ### Prerequisites
 
 The provided code was tested with Python 3.8 and the following libraries:
 SoundFile 0.10.3, mat73 0.58, numpy 1.20.1, scipy 1.6.2, librosa 0.8.1. 
 
-Must also download these RIR databases:
+Must download these RIR databases:
 * TAU SRIR DB: https://zenodo.org/record/6408611
 
-Also download
-* FSD50K https://zenodo.org/record/4060432
-* orchset (could be done using `mirdata`)
+### One-time setup
+
+This data synthesizer uses sound events from the [FSD50K](https://zenodo.org/record/4060432#.ZE7ely2B0Ts) dataset. To download the dataset for first time and to also include music tracks (from the the [FMA dataset](https://github.com/mdeff/fma) dataset) as part of the data synthesizer sound events, do the following under `prepare_fsd50k.py`:
+
+- Change the dataset parameter configuration paths and select `"download": True` to download the FSD50K dataset along with the music FMA dataset.
 
 ```
-pip install mirdata
+PARAM_CONFIG = {
```

Comment:
Ideally this will be pulled from `.yaml` down the line, but this is good for now.